README.md

```diff
@@ -111,6 +111,7 @@ Visit [docs/index.md](./docs/index.md) to see project's documentation.
 
 - English 🇬🇧
 - Polish / Polski 🇵🇱
+- French / Français 🇫🇷
```

Comment:
Since this is an incomplete translation, it worth mentioning it here:
```suggestion
- French / Français 🇫🇷 (work in progress, temporarily disabled)
```